Tests/SwiftDocCTests/Model/ParametersAndReturnValidatorTests.swift

```diff
@@ -196,6 +196,39 @@ class ParametersAndReturnValidatorTests: XCTestCase {
         }
     }
     
+    func testFunctionsThatCorrespondToPropertiesInAnotherLanguage() throws {
+        let (_, _, context) = try testBundleAndContext(named: "GeometricalShapes")
+        XCTAssertEqual(context.problems.map(\.diagnostic.summary), [])
+        
+        // A small test helper to format markup for test assertions in this test.
+        func _format(_ markup: [any Markup]) -> String {
+            markup.map { $0.format() }.joined()
+        }
+        
+        let reference = try XCTUnwrap(context.knownPages.first(where: { $0.lastPathComponent == "isEmpty" }))
+        let node = try context.entity(with: reference)
+        
+        let symbolSemantic = try XCTUnwrap(node.semantic as? Symbol)
+        let swiftParameterNames = symbolSemantic.parametersSectionVariants.firstValue?.parameters
```

Comment:
minor: This looks to be relying on the fact that Swift will be first in the list of variants, which might change in the future, do you think it's worth changing to find the right one?
```suggestion
        let swiftParameterNames = symbolSemantic.parametersSectionVariants.allValues.mapFirst(where: { (trait, variant) -> [Parameter]? in
            guard trait.interfaceLanguage == SourceLanguage.swift.id else { return nil }
            return variant.parameters
        })
```

Reply:
I think either implementation is fine since this is in a test. The behavior of a variant collection is very unlikely to change since there's code that depends on the current behavior.

I made a quick search and we have 37 places in the code that uses one of [these legacy accessors](https://github.com/swiftlang/swift-docc/blob/main/Sources/SwiftDocC/Semantics/Symbol/Symbol.swift#L572-L734) which would break if Swift wasn't the first value.